GitTrends.iOS/Info.plist

```diff
@@ -63,5 +63,10 @@
 	</array>
 	<key>CFBundleVersion</key>
 	<string>30</string>
+	<key>CFBundleLocalizations</key>
+	<array>
+		<string>en</string>
+		<string>nl</string>
+	</array>
```

Comment:
This fixes labels like "Done" etc. on the keyboard, but apparently not the push permissions dialog buttons (see #114). I guess for that you need the localized info.plist things